src/main/java/uk/gov/justice/digital/delius/service/ContactService.java

```diff
@@ -124,6 +123,24 @@ public void addContactForBulkCustodyKeyDateUpdate(final Offender offender, final
                 notesForKeyDatesUpdate(datesAmendedOrUpdated, datesRemoved));
     }
 
+    @Transactional
+    public void addContactForTierUpdate(Long offenderId, LocalDateTime date, String tier, String reason, Staff staff, Team team){
```

Comment:
I know it's a bit hit or miss in community-api, but this class is using `final` parameters so should probably do the same for consistency.

Reply:
Done